src/organizations/OrganizationEntity.js

```diff
@@ -217,7 +236,16 @@ export class OrganizationEntity extends Entity {
 		if (!this._entity || !this._entity.hasLinkByRel(rel)) {
 			return;
 		}
-
 		return this._entity.getLinkByRel(rel).href;
 	}
+
+	getCompletionAction(nowTracked) {
```

Comment:
should we instead have methods: applyCompletionTracking(enabed) and applyDislayProgress(enabled) and perform action here instead of delegating it to the user of the Organization entity ?

Reply:
I suggest `updateCompletionTracking(enabled)`

Reply:
I can move the entirety of the logic to the OrganizationEntity if desired. If so, I like the update option. 

Side note: if this is being done, I do not know how to test for the action being performed.

Reply:
[aa0e3b0](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/216/commits/aa0e3b0982d6400e6144b75673d7fcf954375202) is a compromise to separate the logic of getting the action from performing the action. These functions are underscored and tests are updated to reflect that, but the update versions are there to request the action and perform it.

Reply:
For testing actions, you don't need to actually check the functionality of the update - we assume the API works as expected. Here's a set of tests that can be used as an example for testing an action: https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/master/test/organizations/OrganizationAvailabilityEntity.js#L84

Reply:
Thanks. Added the testing for actions in [ba169f8](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/216/commits/ba169f8b2fc43768a8d5d42bf27d0ec52b03644a).